### PR TITLE
Remove unused searchPath from ProjectionBuilder#aggregationProjection()

### PR DESCRIPTION
**Pull request by matriv**

### CODE REVIEW

File: server/src/testFixtures/java/io/crate/operation/aggregation/AggregationTestCase.java [hunk 1]
```diff
@@ -321,6 +321,7 @@ private List<Row> execPartialAggregationWithDocValues(LuceneReferenceResolver re
                 )
             );
         }
+
```
Comment (mfussenegger) [inline server/src/testFixtures/java/io/crate/operation/aggregation/AggregationTestCase.java:324]: :scream:

**Review by mfussenegger — APPROVED** [review]

**Comment by matriv** [comment]
retest this please

**Comment by matriv** [comment]
retest this please

**Comment by matriv** [comment]
retest this please

**Pull request merged**